### ISSUE TITLE
Accessibility audit: (JAWS) Tabs appear as links but missing from link items list

### ISSUE THREAD

**Issue by jonhurrell** [dwp/design-system#302]
## Describe the issue
Global issue to all views with  examples. 
The tabs 'HTML' and 'Nunjucks' appear a links but aren't included within the link items list in JAWS

## Steps to reproduce the issue
With JAWS on navigate to a view with an example and invoke the link item list. This can be reproduced with Voiceover rotor, link menu.
 
## Actual vs expected behaviour
If the tabs are links they should be included in this list to aid navigation.

## Fix Recommendation
- [ ] Investigate why ours and GDS tab links don't appear in this list.

**Comment by dtaylorbrown** [comment]
I've been looking into this a bit and have found that whilst our tabs do look like links regarding styling and hover/focus states we are using the role="presentation" attribute which takes these out of the accessibility tree so this would be the reason that they aren't showing up in the link list. I have fixed a related issue about the Nunjucks tab appearing in the link list when it shouldn't have been: https://github.com/dwp/design-system/issues/299 

I think the behaviour here is correct and as mentioned in the DAC report, they have also raised this with GDS. It might be a case to slightly re-style the tabs so they don't look so much like links but I'll leave that up for discussion.

**Comment by jonhurrell** [comment]
I guess the question in my mind is, why are we using the role if it takes it out of the tree? Any thoughts here @abbott567

**Comment by jonhurrell** [comment]
To address this given we are bending semantics, I replaced the links within the tab-list-items with buttons. The results keep the functionality but add the buttons as form controls in the rotor which aids navigation. I am nervous that I'm missing something given all the examples I can find only use links bar this https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role

https://user-images.githubusercontent.com/809482/146168072-e2f5a009-fcf8-4c11-a253-71b8783a9969.mov

**Comment by jonhurrell** [comment]
A difference is the keyboard selection, hitting space on the 'links' jumps the page, you have to hit enter to expand/collapse. On the button version, both space and enter do the same action, which feels better?
